src/Engine/Audio/SteamAudioRenderers.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <phonon.h>
+#include "../linking.hpp"
+#define SAMPLE_RATE (44100)
+#define CHUNK_SIZE (SAMPLE_RATE / 10)
+
+/// Class responsible for calculating direct audio.
+class SteamAudioRenderers {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'SteamAudioRenderers' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)